tests/trestle/core/profile_resolver_test.py

```diff
@@ -24,7 +24,10 @@
 from tests import test_utils
 
 from trestle.core import generators as gens
-from trestle.core.profile_resolver import CatalogInterface, ProfileResolver
+from trestle.core.catalog_interface import CatalogInterface
+from trestle.core.profile_resolver import ProfileResolver
+from trestle.core.profile_resolver_merge import Merge
+from trestle.core.profile_resolver_modify import Modify
```

Comment:
I believe this could be a good example to create a new package `profile_resolver` and place `merge`, `modify` there?

Reply:
new package called resolver contains them.  profile_resolver.py remains in core